turbinia/turbiniactl.py

```diff
@@ -387,6 +386,22 @@ def main():
       'default timeframe is 7 days. Please use the -d flag to extend this. '
       'Additionaly, you can use the -a or --all_fields flag to retrieve the '
       'full output containing finished and unassigned worker tasks.')
+  parser_log_collector = subparsers.add_parser(
+      'GCPLogsCollector', help='Collects Turbinia logs from Stack driver.')
```

Comment:
Can we lower case the parser/command name here to match the others too?  Maybe we can shorten it to just `gcplogs` too?

Reply:
Done